### PR TITLE
Added TrueSkill delta chips

### PR DESCRIPTION
**Pull request by ronnigaba**

### CODE REVIEW

File: BoyumFoosballStats/Pages/MatchHistory/Models/MatchHistoryViewModel.cs [hunk 1]
```diff
@@ -36,6 +38,31 @@ public void HandlePlayerClicked(Player player)
         _navigationManager.NavigateTo("/PlayerDashboard/" + player.Id);
     }
 
+    public string GetTrueSkillChange(Player player, string? matchId)
```
Comment (alexandrutatarciuc) [inline BoyumFoosballStats/Pages/MatchHistory/Models/MatchHistoryViewModel.cs:41]: Can this go in the `PlayerAnalysisService` instead? Might be need for other stuff later

File: BoyumFoosballStats/Pages/MatchHistory/Models/MatchHistoryViewModel.cs [hunk 2]
```diff
@@ -36,6 +38,31 @@ public void HandlePlayerClicked(Player player)
...
+                var match = Matches[i];
+                if (!match.Players.Any(x => x.Id == player.Id))
+                {
+                    continue;
+                }
+
+                var prevMatchPlayer = match.Players.Single(x => x.Id == player.Id);
+
+                trueSkillChange =
+                    ((player.TrueSkillRating!.Mean - prevMatchPlayer!.TrueSkillRating!.Mean) * 100).ToString("+#;-#;0");
```
Comment (alexandrutatarciuc) [inline BoyumFoosballStats/Pages/MatchHistory/Models/MatchHistoryViewModel.cs:58]: I have also added some logic to convert true skill. I know that there are already 2 different methods for transforming win rate to percentages... I think we need to do something about that

Comment (alexandrutatarciuc) [reply BoyumFoosballStats/Pages/MatchHistory/Models/MatchHistoryViewModel.cs:58]: You don't have to do anything, about this, just letting you know :D

Comment (ronnigaba) [reply BoyumFoosballStats/Pages/MatchHistory/Models/MatchHistoryViewModel.cs:58]: Yea, I noticed that in your PR - thanks! :)

File: BoyumFoosballStats/Pages/MatchHistory/Models/MatchHistoryViewModel.cs (hunk 1 above)
Comment (ronnigaba) [reply BoyumFoosballStats/Pages/MatchHistory/Models/MatchHistoryViewModel.cs:44]: I moved it!

**Review by alexandrutatarciuc — APPROVED** [review]

**Pull request merged**